update.ts

```diff
@@ -231,7 +231,7 @@ await start(manifest, { plugins: [twindPlugin(twindConfig)] });\n`;
     for (const n of nodes) {
       if (!n.wasForgotten() && Node.isJsxAttribute(n)) {
         const init = n.getInitializer();
-        const name = n.getName();
+        const name = n.getStructure().name;
```

Comment:
I don't think there's a test for this, because just commenting the line out also seems to work. I wonder how important all of this is, because I guess not many people are still using the old style:
```
This project is using an old version of the twind integration. Would you like to
update to the new twind plugin? This will remove the 'class={tw\`border\`}'
boilerplate from your source code replace it with the simpler 'class="border"'.
```

But I did do some due diligence, and logged out the result of `n.getName()` with 17.0.1, and that seems to perfectly match what `n.getStructure().name` is doing with 20.0.0, so I guess these two are the same.

Reply:
Sounds good to me. No idea why it was renamed.